rust-toolchain

```diff
@@ -0,0 +1 @@
+1.64.0
```

Comment:
I don't know if this is strictly necessary. Instead we should probably have some check in CI that verifies some MSRV.

Reply:
Ok so I can see that there is some logic for a MSRV in the `ci.yml` file. From the logs, it would appear that this logic still works as intended (ignoring `rust-toolchain`). If you have the same interpretation of the logs, then I'd say that pinning the rust version is beneficial to make CI deterministic and in line with `cargo test` on developer laptops. It basically makes Rust upgrades explicit on the main branch. (For context, yesterday, the build was broken in CI but not on my laptop.)